orchestra/communication/utils.py

```diff
@@ -0,0 +1,61 @@
+from orchestra.models import CommunicationPreference
+from orchestra.models import StaffBotRequest
+from orchestra.models import StaffingRequestInquiry
+from orchestra.models import StaffingResponse
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+def mark_worker_as_winner(worker, task, required_role_counter,
```

Comment:
make this transactional, or make sure everyone who calls it is transational.

(my worry is two Workers winning at the same time and both of them marking the other as not a winner :))

Reply:
sounds g

Reply:
d